trunk/SUAVE/Methods/Weights/Buildups/eHelicopter/wing.py

```diff
@@ -0,0 +1,248 @@
+# Shear/Moments
+#-------------------------------------------------------------------------------
+
+    Vx = np.append(np.cumsum((D[0:-1]*np.diff(x))[::-1])[::-1],0)   # Drag Shear
+    Vz = np.append(np.cumsum((L[0:-1]*np.diff(x))[::-1])[::-1],0)   # Lift Shear
+    Vt = 0 * Vz                                        # Initialize Thrust Shear
+
+    # Calculate shear due to thrust by adding thrust from each motor to each
+    # analysis point that's closer to the wing root than the motor. Accomplished
+    # by indexing Vt according to a boolean mask of the design points that area
+    # less than or aligned with the motor location under consideration in an
+    # iterative loop
+
+    for i in range(np.size(xMotor)):
+        Vt[x<=xMotor[i]] = Vt[x<=xMotor[i]] + maxThrust
+
+    Mx = np.append(np.cumsum((Vz[0:-1]*np.diff(x))[::-1])[::-1],0)  # Bending Moment
+    My = np.append(np.cumsum(( T[0:-1]*np.diff(x))[::-1])[::-1],0)  # Torsion Moment
+    Mz = np.append(np.cumsum((Vx[0:-1]*np.diff(x))[::-1])[::-1],0)  # Drag Moment
+    Mt = np.append(np.cumsum((Vt[0:-1]*np.diff(x))[::-1])[::-1],0)  # Thrust Moment
+    Mz = np.max((Mz,Mt))    # Worst Case of Drag vs. Thrust Moment
+
+#-------------------------------------------------------------------------------
+# General Structural Properties
+#-------------------------------------------------------------------------------
+
+    seg = []                        # LIST of Structural Segments
+
+    # Torsion
+
+    box = coord                     # Box Initally Matches Airfoil
+    box = box[box[:,0]<=aftWeb[1]]   # Inlcude Only Parts Fwd of Aftmost Spar
+    box = box[box[:,0]>=fwdWeb[0]]   # Include Only Parts Aft of Fwdmost Spar
+    box = box * chord               # Scale by Chord Length
+
+        # Use Shoelace Formula to calculate box area
+
+    torsionArea = 0.5*np.abs(np.dot(box[:,0],np.roll(box[:,1],1))-
+        np.dot(box[:,1],np.roll(box[:,0],1)))
+
+    torsionLength = np.sum(np.sqrt(np.sum(np.diff(box,axis=0)**2,axis=1)))
+
+    # Bending
+
+    box = coord                     # Box Initally Matches Airfoil
+    box = box[box[:,0]<=fwdWeb[1]]  # Inlcude Only Parts Fwd of Aft Fwd Spar
+    box = box[box[:,0]>=fwdWeb[0]]  # Include Only Parts Aft of Fwdmost Spar
+    seg.append(box[box[:,1]>np.mean(box[:,1])]*chord)   # Upper Fwd Segment
+    seg.append(box[box[:,1]<np.mean(box[:,1])]*chord)   # Lower Fwd Segment
+
+    # Drag
+
+    box = coord                     # Box Initally Matches Airfoil
+    box = box[box[:,0]<=aftWeb[1]]  # Inlcude Only Parts Fwd of Aftmost Spar
+    box = box[box[:,0]>=aftWeb[0]]  # Include Only Parts Aft of Fwd Aft Spar
+    seg.append(box[box[:,1]>np.mean(box[:,1])]*chord)   # Upper Aft Segment
+    seg.append(box[box[:,1]<np.mean(box[:,1])]*chord)   # Lower Aft Segment
+
+    # Bending/Drag Inertia
+
+    flapInertia = 0
+    flapLength  = 0
+    dragInertia = 0
+    dragLength  = 0
+
+    for i in range(0,4):
```

Comment:
Use xrange

Reply:
Changed to xrange throughout the project.